@xen-orchestra/web-core/lib/components/menu/MenuList.vue

```diff
@@ -69,15 +78,17 @@ const open = (event: MouseEvent) => {
 
   isOpen.value = true
 
-  nextTick(() => {
-    clearClickOutsideEvent = onClickOutside(menu, () => (isOpen.value = false), {
-      ignore: [event.currentTarget as HTMLElement],
-    })
+  if (!isMobile.value) {
+    nextTick(() => {
+      clearClickOutsideEvent = onClickOutside(menu, () => (isOpen.value = false), {
+        ignore: [event.currentTarget as HTMLElement],
+      })
 
-    placementJs(event.currentTarget as HTMLElement, unrefElement(menu), {
-      placement: props.placement ?? (isParentHorizontal.value ? 'bottom-start' : 'right-start'),
+      placementJs(event.currentTarget as HTMLElement, unrefElement(menu), {
+        placement: props.placement ?? (isParentHorizontal.value ? 'bottom-start' : 'right-start'),
+      })
     })
-  })
+  }
```

Comment:
What do you think of using an early return instead of wrapping all the code inside the `if`?